ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsManagerImpl.scala

```diff
@@ -76,14 +79,14 @@ case class MetricsManager[T](
 
 object MetricsManager {
   def apply[StreamElem](
-      streamName: String,
+      observedMetric: String,
       logInterval: FiniteDuration,
       metrics: List[Metric[StreamElem]],
       exposedMetrics: Option[ExposedMetrics[StreamElem]],
   )(implicit
       system: ActorSystem[SpawnProtocol.Command],
       ec: ExecutionContext,
-  ): Future[MetricsManager[StreamElem]] = {
+  ): Future[MetricsManagerImpl[StreamElem]] = {
```

Comment:
I am wondering if here it is actually better to return interface rather than particular impl?

Reply:
You are right. Oversaw this